src/client/components/atoms/textArea/index.tsx

```diff
@@ -0,0 +1,11 @@
+import classNames from 'classnames';
+import { DetailedHTMLProps, TextareaHTMLAttributes } from 'react';
+import './style.scss';
+
+export type TextAreaProps = DetailedHTMLProps<TextareaHTMLAttributes<HTMLTextAreaElement>, HTMLTextAreaElement>;
+
+const TextArea = ({ className, name, ...props }: TextAreaProps) => (
+  <textarea name={name} className={classNames('text-area', className)} {...props} />

```

Comment:
two things here:

1. Since we are not using our Input here, we dont have the floating label so, let's create a label for the textArea
2. create a block wrapping both with `className="textarea"`
3. only use classNames package for dynamic classes.

```
<div className="text-area__container">
  <label>props.label</label>
  <textarea name={name} className="textarea"} {...props} />
</div>